lua/autorun/sh_waiter.lua

```diff
@@ -42,7 +42,7 @@ end
 local function attendPatron( patronID, patron )
     print( "[CFC Waiter] Attending to patron ID: " .. tostring( patronID ) )
     if patron.attempts >= patron.maxAttempts then
-        print( "[CFC Waiter] Patron ID " .. tostring( patronID ) .. " has reached max attempts! Running onTimeout and removing.." )
+        print( "[CFC Waiter] Patron ID " .. tostring( patronID ) .. " has reached max attempts! Running onTimeout and removing .. " )
```

Comment:
```suggestion
        print( "[CFC Waiter] Patron ID " .. tostring( patronID ) .. " has reached max attempts! Running onTimeout and removing..." )
```

Reply:
Ah, you got him! 
**HE USED SED, HE DIDN'T ACTUALLY DO ANY LEGWORK**